authors/settings.py

```diff
@@ -158,7 +158,10 @@
 REST_FRAMEWORK = {
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
-
+    'DEFAULT_PERMISSION_CLASSES': (
```

Comment:
I don't think we need this.

Reply:
the default classes i don't think we need them too..